upgrades/backend.go

```diff
@@ -193,82 +128,14 @@ func (s stateBackend) AddActionPruneSettings() error {
 	return state.AddActionPruneSettings(s.pool)
 }
 
-func (s stateBackend) AddUpdateStatusHookSettings() error {
-	return state.AddUpdateStatusHookSettings(s.pool)
-}
-
-func (s stateBackend) AddStorageInstanceConstraints() error {
-	return state.AddStorageInstanceConstraints(s.pool)
-}
-
-func (s stateBackend) SplitLogCollections() error {
-	return state.SplitLogCollections(s.pool)
-}
-
-func (s stateBackend) CorrectRelationUnitCounts() error {
-	return state.CorrectRelationUnitCounts(s.pool)
-}
-
-func (s stateBackend) AddModelEnvironVersion() error {
-	return state.AddModelEnvironVersion(s.pool)
-}
-
-func (s stateBackend) AddModelType() error {
-	return state.AddModelType(s.pool)
-}
-
 func (s stateBackend) MigrateLeasesToGlobalTime() error {
```

Comment:
Can be removed as well. There are plenty more of these implementations that are now unused.

Reply:
Removed.